SU2_CFD/src/numerics/turbulent/turb_sources.cpp

```diff
@@ -922,3 +936,40 @@ void CSourcePieceWise_TurbSST::SetPerturbedStrainMag(su2double turb_ke){
   PerturbedStrainMag = sqrt(2.0*PerturbedStrainMag);
 
 }
+
+void CSourcePieceWise_TurbSST::ResidualAxisymmetric(su2double alfa_blended, su2double zeta){
+
```

Comment:
I would define this inline (in the hpp) and let the compiler have some fun trying to re-use some of the stuff that exists in the body of ComputeResidual.
Probably not massive performance gains but they are completely free in this case.

Otherwise everything looks very nice. Are you going to add the regression in this PR?

Reply:
Okay why not... I make it inline 

I was going to add the case in another PR but as you prefer. I reckon there is not much risk in merging this in already since the current state of affairs is no more validated or tested for performance and strictly wrong as it is incomplete. From a little testing I can see only a small effect on the results but convergence seems to be improved

Reply:
Got it, I think it is always best to add it together with the code, while the reference results are fresh in your mind.
If you add it later you might find yourself having to debug again because of some change somewhere else.

Reply:
> since the current state of affairs is no more validated or tested for performance and strictly wrong as it is incomplete. 

It's cases like these (small changes) where unit tests are handy: https://su2code.github.io/docs_v7/Writing-Unit-Tests/

Reply:
Not sure about this but I don't really see the need for unit tests as this bit is structurally very straightforward code and its behaviour is tested in regression now anyway

Reply:
Unit tests detect change rather than correctness.
It is possibly that while fixing a bug a new one is introduced, regression values change, people check a few of them (it is boring work so no one tests all of them, especially if the residuals change very little), people update the regressions...
It has happened before, it will happen again.
With unit tests it is harder for that to happen, depending on the size of the "unit"...

Reply:
First sentence was wrong,  text was edited.

Reply:
Alright I can see how that could happen but can't come up with a good unit test for this part. Keeping this in mind for the future though

Reply:
I agree with @pcarruscag.  I brought them up because "small changes that aren't complete" is a perfect use case for unit tests.  Unit tests can be added to verify correctness before code is ready for a complete regression test.